### DISCUSSION TITLE
This library is so good

### DISCUSSION THREAD

**Discussion by kenchoong** [kirillzyusko/react-native-keyboard-controller#44] [General]
Just create this thread to thank you for your work. 

This library is too awesome

**Comment by kirillzyusko** [comment]
@kenchoong thank you for the warm words 😊

**Reply by yzalvov** [reply]
@kirillzyusko much respect my friend! 👏🏻👏🏻👏🏻

NOTHING worked on Android (we tried a bunch of solutions, hacks, 3d party libraries)... Your `useReanimatedKeyboardAnimation` in pair with `useAnimatedStyle` of the `react-native-reanimated` works like a charm, wow

**Reply by kirillzyusko** [reply]
@yzalvov wow, glad to hear it 😊

**Comment by Bowlerr** [comment]
just wanted to add another thank u for your service ✨💛

Looking forward to testing this soon to see if it solves my current issues with the native keyboard avoiding view we have

**Comment by LagoonProject** [comment]
After struggling for hours during months to handle keyboard inputs in RN, this library feels like a godsend, it just works and brings peace of mind. It should be by default in RN. Huge thank you to the developer, your work is much appreciated.

**Comment by qstrauss** [comment]
Awesome, a Huge Thank You

**Comment by kimangell** [comment]
Just came here wanting to create a similar post. I'll just add my praises here. Awesome work, thanks to everyone involved! ❤️‍🔥

**Comment by KiwiKilian** [comment]
After fighting for years with the default `KeyboardAvoidingView` this library really was a relief. Thanks for your maintenance!

**Comment by danielcrk** [comment]
Wanting to add to this. After iOS26, this library has had its share of issues and bugs, but Kirill has persisted where most of us would have given up. I'm so glad we have KeyboardController, I honestly don't know if React Native would even be a viable option for me without it.
